### PR TITLE
[FixaMinGata] Incremental updates

### PR DESCRIPTION
**Pull request by jonkri**
Please check the following:

- [x] Whether this PR should include changes to any documentation, or the FAQ;
- [x] All cobrand-specific commits start their commit message with the cobrand in square brackets;
- [ ] Is new functionality tested? CodeCov will warn you about the diff coverage, but won’t complain about e.g. new files;
- [x] Will cobrand-specific changes require additional work to ensure consistent behaviour on www.fixmystreet.com? 
- [x] Are the changes tested for accessibility?
- [ ] Have you updated the changelog? If this is not necessary, put square brackets around this: [skip changelog]

Please check the contributing docs, and describe your pull request here.
Screenshots or GIF animations (using e.g. LICEcap) may be helpful.

Please include any issues that are fixed, using "fixes" or "closes" so that
they are auto-closed when the PR is merged.

Thanks for contributing!

### CODE REVIEW

**Comment by jonkri** [comment]
Please let us know if testing is necessary for this pull request. Thanks!

File: templates/web/fixamingata/front/tips.html [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+     </video>
 </div>
+<script>
+    // Lose focus on pc element to avoid a virtual keyboard problem on Android
+    var fmgVideo = document.getElementById('fmg-video'),
+        pc = document.getElementById('pc');
+
+    function blurPc() {
+        if (pc.blur) {
+            console.log('blur');
```
Comment (dracos) [inline templates/web/fixamingata/front/tips.html:16]: Probably don't want this line? :)

**Review by dracos — COMMENTED** [review]
One left in console.log is all, looks like :)

File: templates/web/fixamingata/front/tips.html (hunk 1 above)
Comment (jonkri) [reply templates/web/fixamingata/front/tips.html]: Thank you for spotting this! I have removed the call and rebased.

**Comment by jonkri** [comment]
I just fixed a merge conflict in `locale/sv_SE.UTF-8/LC_MESSAGES/FixMyStreet.po`. From what I can tell, this should be ready to be merged. 😊

**Comment by dracos** [comment]
Great - can you either rebase on top of current mysociety master, or give me permission to edit this PR so that I can do that?

**Comment by jonkri** [comment]
@dracos: I have rebased on top of mySociety's master branch. Thanks! 😊

**Pull request merged**